### PR TITLE
Add release notes for Relativity.Transfer.SDK 2.0.0

### PR DESCRIPTION
**Pull request by KamilKalitaRel**

### CODE REVIEW

**Review by KlaudiuszBryjaRelativity — APPROVED** [review]

File: Relativity.Transfer.SDK.md [hunk 1]
```diff
@@ -4,6 +4,33 @@
...
+- Introduced `OvewritePolicy` in `OptionsBase` for implementing policies to handle file skipping and overwriting at the destination.
+- `AzureTransferJobClient` now supports uploading a list of files to the fileshare.
+- Added the ability to specify the location of journal files in transfer options.
+
+#### Improvements:
+- Improved handling of I/O errors on Windows for better reliability and troubleshooting.
+- Transfer SDK will now automatically remove journal files older than 28 days to optimize storage.
+
+#### Other:
+- Support for legacy fileshares has been discontinued since all clients have successfully migrated to the new storage.
```
Comment (GrzegorzSidwa) [inline Relativity.Transfer.SDK.md:26]: Shouldn't it be part of breaking change section?

File: Relativity.Transfer.SDK.md [hunk 2]
```diff
@@ -4,6 +4,33 @@
...
+- `AzureTransferFullPathClientBuilder` and `AzureTransferJobClientBuilder` now require a workspace context, which can be specified by the workspace's ID or using Staging Explorer privileges.
+- Cancelled directory transfers will no longer throw `OperationCancelledException`. Instead, a result with a "Cancelled" state will be returned.
+- If the client's implementation of `IRelativityAuthenticationProvider` throws an exception, the SDK will now throw a `FailedToAcquireCredentialsException`.
+
+#### New Features:
+- Introduced `OvewritePolicy` in `OptionsBase` for implementing policies to handle file skipping and overwriting at the destination.
+- `AzureTransferJobClient` now supports uploading a list of files to the fileshare.
+- Added the ability to specify the location of journal files in transfer options.
+
+#### Improvements:
```
Comment (GrzegorzSidwa) [inline Relativity.Transfer.SDK.md:21]: I would suggest to combine improvements and new features into one section. For example journal removal is feature for me ;)

File: Relativity.Transfer.SDK.md (hunk 1 above)
Comment (KlaudiuszBryjaRelativity) [reply Relativity.Transfer.SDK.md]: Done

File: Relativity.Transfer.SDK.md (hunk 2 above)
Comment (KlaudiuszBryjaRelativity) [reply Relativity.Transfer.SDK.md]: Done

**Review by GrzegorzSidwa — APPROVED** [review]

**Pull request merged**